test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
```

Comment:
Remember, that we are using `RSpec` for automated tests in our application.
(I know that this file was generated automatically, but you should change its structure to be `RSpec` compatible)